src/ui/views/menu.coffee

```diff
@@ -432,6 +432,10 @@ templateWindow = (viewstate) -> [
 templateMenu = (viewstate) ->
     [
         {
+            label: ''
+        }
+        {
```

Comment:
additionally

could you add as a comment the original issue number (#556) as to trace back the change in the code

looking at it without context it could look like a strange menu to have :)